hpx/runtime/threads/coroutines/detail/context_linux_x86.hpp

```diff
@@ -93,16 +93,17 @@ namespace hpx { namespace threads { namespace coroutines
             return u.t;
         }
 
-        template<typename T>
-        HPX_FORCEINLINE void trampoline(T* fun);
+//         template<typename T>
+//         HPX_FORCEINLINE void trampoline(void* fun);
```

Comment:
Can this be removed? Shouldn't then the function below be marked FORCEINLINE instead?